app/default/props.conf

```diff
@@ -19,24 +19,31 @@ TRANSFORMS-source_type = heroku_system_source_type, heroku_requests_source_type,
 # Do not index some high volume low value messages.
 TRANSFORMS-heroku_requests_null_queue = heroku_requests_null_queue
 
-[heroku_system]
+[heroku:system]
```

Comment:
Renaming the source types to use a `:` convention, based on the tips in https://www.splunk.com/en_us/blog/tips-and-tricks/sourcetypes-whats-in-name.html.

Reply:
I'd also like to explore renaming `heroku` to `heroku:app` in another pull request. Keep `heroku` as the "source type router" and add a new one for `heroku:app`, might be possible but need to test it.

Reply:
I like semi-colons in the source types 👍 if we're making this change (and the `heroku` -> `heroku:app` one) then it'd be great to do so before we roll out to too many Customer Products repos. I'd rather not have to update migration guides and dashboards too many times

Reply:
Yep I see this as the final get naming right before the big adoption is there 😂

Reply:
Split out the source type renaming into #41.